src/wmtk/operations/EdgeCollapse.cpp

```diff
@@ -49,9 +51,10 @@ std::vector<Simplex> EdgeCollapse::unmodified_primitives(
 ////////////////////////////////////
 std::vector<Simplex> EdgeCollapse::execute(TetMesh& mesh, const Simplex& simplex)
 {
-    Accessor<long> accessor = hash_accessor();
-    auto return_data = mesh.collapse_edge(simplex.tuple(), accessor);
-    return {Simplex::vertex(return_data.m_output_tuple)};
+    auto return_data = operations::utils::multi_mesh_edge_collapse(mesh, simplex.tuple());
+    const operations::tet_mesh::EdgeOperationData& my_data = return_data.get(mesh, simplex);
+    deleted_edges = my_data.deleted_simplex_tuples(mesh, PrimitiveType::Edge);
```

Comment:
I thought edgecollapse shouldn't store state - did we agree to have this around for this round?

Reply:
yes this is not used. I'll remove it